numpy/core/src/multiarray/shape.c

```diff
@@ -680,9 +680,9 @@ PyArray_SwapAxes(PyArrayObject *ap, int a1, int a2)
 NPY_NO_EXPORT PyObject *
 PyArray_Transpose(PyArrayObject *ap, PyArray_Dims *permute)
```

Comment:
Why is it correct that the list of axes be int? Int may sufficient, but I doubt it is necessary.